core/src/main/resources/jenkins/agents/CloudSet/_new.jelly

```diff
@@ -33,6 +33,9 @@ THE SOFTWARE.
     <l:main-panel>
       <f:form method="post" action="doCreate" name="config" class="jenkins-form">
         <h1>${%New cloud}</h1>
+        <f:entry title="${%Name}" field="cloudName">
```

Comment:
the name field gets sent to `_new.jelly` 
https://github.com/jenkinsci/jenkins/blob/3059161906e63eb3be7a9f7115a70437818b9116/core/src/main/java/jenkins/agents/CloudSet.java#L228
However, if we remove the name field from downstream plugins in their `config.jelly`, the name becomes completely lost when the cloud object is first created in `doDoCreate`.

Another alternative I can think of is storing the name value in `CloudSet`

Reply:
why is it cloudName here?  I thought the proper name was `name`?

Reply:
whoops, you're right, `cloudName` also happened to be passed along in the form data so it was working in my tests.

Reply:
I think `cloudName` is also unnecessary as it is created as a special case for `ec2-plugin`. Removing in https://github.com/jenkinsci/ec2-plugin/pull/888

Reply:
nevermind, see: https://github.com/jenkinsci/jenkins/pull/8310#discussion_r1322315141